packages/auth-providers/supabase/api/package.json

```diff
@@ -24,13 +24,15 @@
   },
   "dependencies": {
     "@babel/runtime-corejs3": "7.24.1",
+    "@redwoodjs/vite": "workspace:*",
     "core-js": "3.36.1",
     "jsonwebtoken": "9.0.2"
   },
   "devDependencies": {
     "@babel/cli": "7.24.1",
     "@babel/core": "^7.22.20",
     "@redwoodjs/api": "workspace:*",
+    "@redwoodjs/vite": "workspace:*",
```

Comment:
Don't think we need duplicated devDependency. I can't see why we have a dependency on this atm either.